config/langs.toml

```diff
@@ -894,6 +894,32 @@ foreach arg $argv {
 }
 '''
 
+[TeX]
```

Comment:
Add
```toml
experiment = 999
```
Where 999 is the issue ID for the TeX lang (it might need creating) like we do for holes.